src/configServer.js

```diff
@@ -7,7 +7,8 @@ import { findAvailablePort, getConfig } from './utils'
 export async function startConfigServer () {
   const config = getConfig()
   // scan a range
-  const port = await findAvailablePort(8000)
+  let port = await findAvailablePort(8000);
+  port = process.env.PORT || port;
```

Comment:
You need to remove the semicolons for the linter to get happy. :)

Reply:
in your working copy of the repo, you can also just run
`yarn test --fix`
to automatically fix all linting issues.